client/src/redux-toolkit/features/users/userSlice.js

```diff
@@ -8,9 +9,9 @@ const initialState = {
 
 export const createNewUser = createAsyncThunk('user/createNewUser', async (newUserObj) => {
     try {
-        await axios.post("http://localhost:5500/user/createNewUser", newUserObj);
+        await axios.post(`${USER_BASE_URL}/createNewUser`, newUserObj);
     } catch (error) {
-        return error.response.data.error;
+        return  error.response.data
     }
 })
```

Comment:
this has nothing to do with redux, don't use asyncThunk, move this api call in other file